indico/MaKaC/common/cache.py

```diff
@@ -472,7 +482,30 @@ def delete(self, key):
         return 1
 
 
+class MemcachedCacheClient(CacheClient):
+    """Memcached-based cache client"""
+
+    @staticmethod
```

Comment:
it doesn't use anything from the instance but imho adding `@staticmethod` just because it *could* be static is a bit weird